R/class.R

```diff
@@ -88,7 +89,8 @@ validate_messydate <- function(x) {
 
   if (any(grepl("!|\\(|\\)|\\+|\\=|\\/|,|;|>|<|_|\\^|'|&|\\$|#", values))) {
     stop(
-      "Messy dates can only consist of numbers and some special symbols: []{}..X%?~",
+      "Messy dates can only consist of numbers and
```

Comment:
Is this `{messydates}` the package name or "messy dates" the concept or `messydt` the object class?